dysnix/bsc/templates/statefulset.yaml

```diff
@@ -0,0 +1,161 @@
+          - --syncmode={{ .Values.bsc.syncmode }}
+          - --gcmode={{ .Values.bsc.gcmode }}
+          - --maxpeers={{ .Values.bsc.maxpeers }}
+          - --cache={{ .Values.bsc.cache }}
+          - --snapshot=false
+          - --port={{ .Values.service.p2pPort0 }}
+          {{- if and .Values.externalLBp2p .Values.externalLBp2pIP }}
+          - --nat=extip:{{- .Values.externalLBp2pIP -}}
+          {{- end }}
+        workingDir: "{{ .Values.bsc.base_path }}"
+        resources:
+          {{- toYaml .Values.resources | nindent 10 }}
+        ports:
+          {{- range $val := .Values.service.ports }}
+        - containerPort: {{ $val.port }}
+          name: "{{ $val.name }}"
+          protocol: {{ $val.protocol | default "TCP" }}
+          {{- end }}
+        volumeMounts:
+        - name: bsc-config
+          mountPath: /config
+        - name: scripts
+          mountPath: /scripts
+        - name: bsc-pvc
+          mountPath: {{ .Values.bsc.base_path }}
+        livenessProbe:
+          exec:
+            command:
+            - /bin/sh
+            - /scripts/check_node_health.sh
+            - http://127.0.0.1:{{ .Values.service.rpcPort }}
+            - "300"
+            - last_synced_block.txt
+          initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds }}
+          periodSeconds: {{ .Values.livenessProbe.periodSeconds }}
+          timeoutSeconds: {{ .Values.livenessProbe.timeoutSeconds }}
+          successThreshold: {{ .Values.livenessProbe.successThreshold }}
+          failureThreshold: {{ .Values.livenessProbe.failureThreshold }}
+        {{- if .Values.readinessProbe }}
+        readinessProbe:
+          exec:
+            command:
+            - /bin/sh
+            - /scripts/check_node_readiness.sh
+            - --timestamp-distinct
+            - "300"
+          initialDelaySeconds: {{ .Values.readinessProbe.initialDelaySeconds }}
+          periodSeconds: {{ .Values.readinessProbe.periodSeconds }}
+          timeoutSeconds: {{ .Values.readinessProbe.timeoutSeconds }}
+          successThreshold: {{ .Values.readinessProbe.successThreshold }}
+          failureThreshold: {{ .Values.readinessProbe.failureThreshold }}
+        {{- end }}
+      - name: logger
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        command: [ "tail","-F","{{ .Values.bsc.base_path }}/bsc.log" ]
+        workingDir: "{{ .Values.bsc.base_path }}"
+        volumeMounts:
+          - name: bsc-pvc
+            mountPath: {{ .Values.bsc.base_path }}
+            readOnly: true
+      initContainers:
+      {{- if .Values.init_genesis }}
+      - name: init
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        command: ["/bin/sh", "-c", "wget -O /tmp/mainnet.zip
+        https://github.com/binance-chain/bsc/releases/download/v1.1.0-beta/mainnet.zip &&
```

Comment:
we need to template `v1.1.0-beta`, through `appVersion` f.e.

Reply:
Fixed.